game/lib/widget/leaderboard_cell.dart

```diff
@@ -34,6 +34,13 @@ Widget leaderBoardCell(
     height: 0,
   );
 
+// Helper Function with ellipses functionality such that we can set max character length length/cutoff
+  String truncateWithEllipsis(String text, int maxLength) {
```

Comment:
I think it'll be better to truncate more characters if the original text just exceeds maxLength. This will make users less likely to be confused by unnecessary truncation of short texts.
For example: if the text is "I like that", this original implementation will truncate it into "I like tha...", but when users click to expand this truncation, they will only see one character "t" being truncated, and they might be sad of seeing only one letter.